### DISCUSSION TITLE
Unable to connect to Create 3 from VMWare Workstation Player (Ubuntu 20.04)

### DISCUSSION THREAD

**Discussion by fiazmohamed** [iRobotEducation/create3_docs#67] [Questions]
Followed the instructions at https://iroboteducation.github.io/create3_docs/setup/ubuntu2004/ to install ros2 (Ubuntu 20.04 via VMware Workstation Player) and connect to create3. I'm confused about the part where she (https://edu.irobot.com/create3-setup; timestamp 17:55) switches to WiFi. How do I do this on VMware Workstation? This seems to work fine on Parallels (Ubuntu 20.04 on Mac host) but I need this to run on a VMware Workstation (on Windows 11 host).

**Comment by shamlian** [comment]
I've never used VMWare Player, [but here are their docs](https://docs.vmware.com/en/VMware-Workstation-Player-for-Windows/16.0/com.vmware.player.win.using.doc/GUID-C82DCB68-2EFA-460A-A765-37225883337D.html); I suspect you'll want to select "Bridged."

**Reply by gy2256** [reply]
I actually have trouble to get it working with Parallels with M1 MacBook Pro 14. The Parallel VM has Ubuntu 20.04 with ROS Galactic, the network is set to Bridged: wifi. I am able to use the ROS demo talker and listener between the VM and a separate Linux computer running another ROS, so the connection is fine. But I cannot get any ros topics from the robot when I type ros2 topic list in the VM. My other computer (running Linux) is able to see all the topics and send commands to the robot. Not sure if it's the M1 issue or Parallel desktop issue.
